packages/xo-server-perf-alert/src/index.js

```diff
@@ -596,23 +611,23 @@ ${entry.listItem}`
           continue
         }
 
-        const raiseAlarm = alarmId => {
-          // sample XenCenter message:
-          // value: 1.242087 config: <variable> <name value="mem_usage"/> </variable>
-          this._xo
-            .getXapi(entry.object.uuid)
-            .call(
-              'message.create',
-              'ALARM',
-              3,
-              entry.object.$type,
-              entry.object.uuid,
-              `value: ${entry.value.toFixed(
-                1
-              )} config: <variable> <name value="${
-                monitor.variableName
-              }"/> </variable>`
-            )
+        const raiseAlarm = _alarmId => {
```

Comment:
Why this change?

Reply:
to remove a warning

Reply:
Which warning?

Reply:
"unused parameter" 
![Capture d’écran 2020-05-18 à 14 08 27](https://user-images.githubusercontent.com/46941/82211405-13913a80-9911-11ea-88e3-67685ed60a49.png)

Reply:
Why not simply remove this parameter then,

Reply:
to not force the next reader to follow the chain of calls to know what will be passed there.